### ISSUE TITLE
Future enhancement idea: Easy way of making custom repositories

### ISSUE THREAD

**Issue by Daniel15** [yarnpkg/yarn#773]
One of the things I love about Linux repositories (such as Debian and RPM repositories) is that it just consists of plain files and there's no custom server-side component at all. There's a few files that contain metadata about all the packages, along with the actual binaries themselves (eg. `.deb` for Debian, `.rpm` for CentOS). This makes it super easy to host a repository as long as the metadata is built correctly. You can even host a package repo on a plain basic webhost such as Github Pages.

It would be awesome to have something like this for Yarn too, a simple way of making a custom package server. "Throw these files onto a server somewhere" is a much easier deployment strategy than running some custom server software and having to worry about keeping it up-to-date with security patches etc.

Related: #547 (support for multiple servers), #606 (number of stars on this issue means a lot of people are using custom servers already)

**Comment by gsklee** [comment]
@Daniel15 If you're still looking to pursue this idea, the scale of this feature request means that it needs to go through the [proper RFC process](https://github.com/yarnpkg/rfcs).

**Comment by Daniel15** [comment]
Yeah... This issue was created before the RFC process was finalized, hence why it's in the repo.

**Comment by gsklee** [comment]
Well in that case, is it safe to say that we can close this issue for now? I'm helping out with #3474 :-p
